auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -259,8 +259,8 @@ export default defineComponent({
     }
 
     /** Navigation handler for entities dashboard. */
-    const goToDashboard = (businessIdentifier: string): void => {
-      ConfigHelper.addToSession(SessionStorageKeys.BusinessIdentifierKey, businessIdentifier)
+    const goToDashboard = (businessIdentifier: string, nrNumber?: string): void => {
```

Comment:
Revert, I don't think we need this? There's already some code that cleans this up